src/state/ui/reducer.js

```diff
@@ -20,8 +26,13 @@ export const isPanelOpen = (state = false, { type, isShowing }) =>
 export const selectedNoteId = (state = null, { type, noteId }) =>
     (SELECT_NOTE === type ? noteId : state);
 
+export const filterName = (state = 'all', { type, filterName }) => {
+    return SET_FILTER === type ? filterName : state;
```

Comment:
I'm 100% okay with this, but it does break the pattern from its neighbors and doesn't have too. either way is great.

```js
export const filterName = ( state = 'all', { type, filterName } ) =>
	(SET_FILTER === type ? filterName : state);
```